src/sprite_battler.cpp

```diff
@@ -259,7 +259,7 @@ void Sprite_Battler::CreateSprite() {
 
 	SetX(battler->GetBattleX());
 	SetY(battler->GetBattleY());
-	SetZ(battler->GetBattleY()); // Not a typo
+	SetZ(PriorityBattle + battler->GetBattleY()); // Not a typo
```

Comment:
Yeah, I understand what the comment means, but in my opinion should be reworded to make it better / constructive (and more clear for new contributors).

-
`//TODO: This is wrong` is a bad comment.
`//TODO: Needs foo and bar to work correctly` is a better comment.